assets/aml-benchmark/components/dataset-downloader/spec.yaml

```diff
@@ -13,7 +13,9 @@ inputs:
     optional: True
   configuration:
     type: string
-    description: If a specific sub-dataset of the dataset to download, specify the configuration name; specify 'all' to download all configurations. Else, leave it null.
+    description: >-
```

Comment:
@amangupta26 , just curious why this PR is showing changes in `assets/aml-benchmark` directory? It seems these changes are already merged in the main branch. cc: @iamrk04